tests/app/helpers/conftest.py

```diff
@@ -192,3 +198,28 @@ def expected_footer_nisra_theme():
             "alt": "NISRA - Northern Ireland Statistics and Research Agency",
         },
     }
+
+
+@pytest.fixture()
+def session_data():
+    return SessionData(
+        tx_id="123",
+        schema_name="test_checkbox",
+        display_address="68 Abingdon Road, Goathill",
+        period_str=None,
+        language_code="cy",
+        launch_language_code="en",
+        survey_url=None,
+        ru_name=None,
+        ru_ref=None,
+        submitted_at=datetime.now(timezone.utc).isoformat(),
+        response_id="321",
+        case_id="789",
+    )
+
+
+@pytest.fixture()
+def session_store(session_data):
```

Comment:
This is also existing [here](https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/tests/app/conftest.py#L73)

Reply:
Done.